composer.json

```diff
@@ -14,6 +14,11 @@
             "email": "taylor@laravel.com"
         }
     ],
+    "conflict": {
+        "mockery/mockery": "<1.3.1|>=2.0",
```

Comment:
I'm still really not sure if we actually want these in the framework package. Maybe just on the testing component.

Reply:
If so, then it wouldn't prevent installation incompatible version at framework level. 

But it does make sense to have it in testing composer.json as well and move `mockery` and `phpunit` to suggest.